### ISSUE TITLE
Discord RPC Integration

### ISSUE THREAD

**Issue by LeviSnoot** [LeviSnoot/FNFest-Status#4]

**Comment by LeviSnoot** [comment]
Trying to leverage `pypresence` for this but I just can't get it to actually display a presence. No errors or anything. Will continue attempting to get this working but if someone with a good understanding of Discord Rich Presence has any suggestions, feel free to pitch in.

**Comment by LeviSnoot** [comment]
As a small update, I believe there is an issue on my end, because the scrap code I've written both with node.js and python seems to work, it's just not showing up as an activity/RPC on Discord _for me_. 

I've even tried diving into C++ and looking at Discord's Game SDK (VERY unfamiliar territory for me) in order to make this happen, but without a way to reliably know if it's working, I can't develop a solution for it. Game processes (like Fortnite and other games) show up as an activity just fine, so it's not a matter of me having incorrect client settings on Discord, and all the debugging I've done shows that the script is connecting successfully to Discord's IPC pipe. Still I'm just not seeing anything in the Discord client itself.

This project was initially intended to just be a Discord RPC for Fortnite Festival, but as I saw more potential, I decided to widen the scope. The plan was always to integrate the RPC functionality though, but as I've tried getting this to happen for weeks at this point without luck, I don't know if it will be possible unfortunately.
